pkg/operator/ceph/cluster/nodedaemon/reconcile.go

```diff
@@ -248,7 +249,9 @@ func (r *ReconcileNode) createOrUpdateNodeDaemons(node corev1.Node, tolerations
 			logger.Debugf("crash collector successfully reconciled for node %q. operation: %q", node.Name, op)
 		}
 	}
-	if cephVersion.IsAtLeast(MinVersionForCephExporter) && !cephCluster.Spec.Monitoring.MetricsDisabled {
+	exporterSockFilePath := sockDir + "/ceph-client.ceph-exporter.asok"
```

Comment:
```suggestion
	exporterSockFilePath := sockDir + "/ceph-client.ceph-exporter.asok"
```

I read somewhere that using `fmt.Sprintf` is better option for string concatenation